src/components/study-lateral-tool-bar.js

```diff
@@ -0,0 +1,195 @@
+import LateralToolbar from './lateral-toolbar';
+import NetworkExplorer from './network/network-explorer';
+import NodeEditor from './graph/menus/node-editor';
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import { StudyDrawer, ToolDrawer } from './study-drawer';
+import NetworkModificationTreePane from './network-modification-tree-pane';
+
+export const drawerExplorerWidth = 300;
+export const nodeEditorWidth = 400;
+export const drawerToolbarWidth = 48;
+export const drawerNetworkModificationTreeWidth = 400;
+
+const useStyles = makeStyles((theme) => ({
+    drawerExplorer: {
+        width: drawerExplorerWidth,
+        transition: theme.transitions.create('margin', {
+            easing: theme.transitions.easing.sharp,
+            duration: theme.transitions.duration.leavingScreen,
+        }),
+        // zIndex set to be below the loader with overlay
```

Comment:
maybe those kind of comments aren't up to date ? are they ?

Reply:
as said before, we can still have overlays